js/app.js

```diff
@@ -60,9 +86,60 @@ function highlightCurrentDate() {
   }
 }
 
+function removeHighlight() {
+  var current_months = document.querySelectorAll('.current-month');
+
+  for (var i = 0; i < current_months.length; i++) {
+    current_months[i].classList.remove('current-month');
+  }
+}
+
+function notifyContactBirthday(contact) {
+  var notification = new Notification(navigator.mozL10n.get('app_title'), {
+    body: contact.name + ' (' + bdayAge(contact) + ')',
+    icon: '/img/icons/icon-128.png'
+  });
+
+  notification.onclick = function () {
+    openContact(contact.id);
+    notification.close();
+  };
+}
+
+function addAlarm(contact) {
+  var alarmDate = new Date;
+  alarmDate.setMonth(contact.bday.getMonth());
+  alarmDate.setDate(contact.bday.getDate());
+  alarmDate.setHours(0);
```

Comment:
I was thinking we could add a settings panel where the user could choose the time he wants.

> Le 3 nov. 2015 à 21:43, Pierre Haessig <notifications@github.com> a écrit :
> 
> In js/app.js:
> 
> > +  var notification = new Notification(navigator.mozL10n.get('app_title'), {
> > +    body: contact.name + ' (' + bdayAge(contact) + ')',
> > +    icon: '/img/icons/icon-128.png'
> > +  });
> > +
> > +  notification.onclick = function () {
> > +    openContact(contact.id);
> > +    notification.close();
> > +  };
> > +}
> > +
> > +function addAlarm(contact) {
> > +  var alarmDate = new Date;
> > +  alarmDate.setMonth(contact.bday.getMonth());
> > +  alarmDate.setDate(contact.bday.getDate());
> > +  alarmDate.setHours(0);
> Thinking about this, maybe for people keeping their phones turned on during the night, it would be "softer" to either:
> 
> notify in the morning (8am ?)
> or the late afternoon of the previous day (6pm ?)
> —
> Reply to this email directly or view it on GitHub.
>